code-buddy.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/index.css" />
+    <title>Student Companion App</title>
+  </head>
+  <body>
+    <header class="headersec">
+      <img
+        class="headersec__background"
+        src="/icons/background_shape_1.svg"
+        alt=""
+      />
+      <h1 class="headersec__title">Code Buddys</h1>
+      <h3 class="headersec__subtitle">on Monday &#8212; 18.07.2020</h3>
+    </header>
+
+    <main class="app-grid">
+      <section class="teams margin-t34">
```

Comment:
It is a bit confusing that the class name of this component is `teams` as we are in `code-buddy.html`. I know that the styles for both components are almost identical, however I personally would separate them and distinguish those two.
```suggestion
      <section class="code-buddy margin-t34">
```